### PR TITLE
KARAF-4673: Rename karaf.shutdown.pid.file to karaf.pid.file

### PR DESCRIPTION
**Pull request by lburgazzoli**

### CODE REVIEW

File: assemblies/features/base/src/main/filtered-resources/resources/etc/config.properties [hunk 1]
```diff
@@ -164,6 +164,11 @@ karaf.startlevel.bundle=80
 karaf.shutdown.port.file=${karaf.data}/port
```
Comment (jbonofre) [inline assemblies/features/base/src/main/filtered-resources/resources/etc/config.properties]: It makes sense to remove the `karaf.shutdown.port.file` as it's no more used.

File: main/src/main/java/org/apache/karaf/main/ConfigProperties.java [hunk 2]
```diff
@@ -130,6 +130,8 @@
     private static final String KARAF_SHUTDOWN_COMMAND = "karaf.shutdown.command";
 
     private static final String KARAF_SHUTDOWN_PID_FILE = "karaf.shutdown.pid.file";
```
Comment (jbonofre) [inline main/src/main/java/org/apache/karaf/main/ConfigProperties.java]: I would remove completely `KARAF_SHUTDOWN_PID_FILE`.

File: assemblies/features/base/src/main/filtered-resources/resources/etc/config.properties (hunk 1 above)
Comment (lburgazzoli) [inline assemblies/features/base/src/main/filtered-resources/resources/etc/config.properties]: I will

File: main/src/main/java/org/apache/karaf/main/ConfigProperties.java (hunk 2 above)
Comment (lburgazzoli) [inline main/src/main/java/org/apache/karaf/main/ConfigProperties.java]: I left it as per backward compatibility

Comment (lburgazzoli) [inline main/src/main/java/org/apache/karaf/main/ConfigProperties.java]: should I remove it and ditch backward compatibility ?

Comment (jbonofre) [inline main/src/main/java/org/apache/karaf/main/ConfigProperties.java]: As you fallback to it in the property, it makes sense to keep it indeed.

**Comment by lburgazzoli** [comment]
@jbonofre reworked

**Comment by jbonofre** [comment]
The changes breaks `bin/stop`:

```
./stop
/home/jbonofre/Workspace/karaf/assemblies/apache-karaf/target/apache-karaf-4.1.0-SNAPSHOT/lib/boot/jna-4.2.2.jar:/home/jbonofre/Workspace/karaf/assemblies/apache-karaf/target/apache-karaf-4.1.0-SNAPSHOT/lib/boot/jna-platform-4.2.2.jar:/home/jbonofre/Workspace/karaf/assemblies/apache-karaf/target/apache-karaf-4.1.0-SNAPSHOT/lib/boot/org.apache.karaf.diagnostic.boot-4.1.0-SNAPSHOT.jar:/home/jbonofre/Workspace/karaf/assemblies/apache-karaf/target/apache-karaf-4.1.0-SNAPSHOT/lib/boot/org.apache.karaf.jaas.boot-4.1.0-SNAPSHOT.jar:/home/jbonofre/Workspace/karaf/assemblies/apache-karaf/target/apache-karaf-4.1.0-SNAPSHOT/lib/boot/org.apache.karaf.main-4.1.0-SNAPSHOT.jar:/home/jbonofre/Workspace/karaf/assemblies/apache-karaf/target/apache-karaf-4.1.0-SNAPSHOT/lib/boot/org.osgi.core-6.0.0.jar
Unable to find port...
```

Let me check what the Main is doing during the stop.

**Comment by jbonofre** [comment]
As part of the other Jira, I would remove the shutdown port to use pid instead (already started something on that).

**Comment by lburgazzoli** [comment]
@jbonofre should I do something ?

**Comment by jbonofre** [comment]
@lburgazzoli I gonna work on it. No worries.

**Comment by lburgazzoli** [comment]
@jbonofre great, if you need me to amend anything on this pr let me know

**Comment by lburgazzoli** [comment]
@jbonofre do you need anything on my side on this ?

**Comment by jbonofre** [comment]
Let me resume my work on this. I keep you posted. Sorry for the delay.

**Comment by lburgazzoli** [comment]
No worries, just running my PRs gc task ;)

**Pull request closed without merging**